### PR TITLE
:coin: sfx + delete :tennis:  + separate coins collected in level vs customer wallet balance

### PR DESCRIPTION
**Pull request by norgeous**

### CODE REVIEW

**Comment by norgeous** [comment]
there is still a problem with keys overlapping / animations being loaded more than once?

Screenshot from 2023-11-26 11-49-23

File: src/objects/Coin.ts [hunk 1]
```diff
@@ -21,10 +21,10 @@ const entityConfig: EntityConfigType = {
   collisionCategory: CollisionCategories.coin,
   animations: [
     {
-      animationKey: 'spin',
+      animationKey: 'idle',
       fps: 12,
       start: 0,
-      end: 5,
+      end: 4,
```
Comment (norgeous) [inline src/objects/Coin.ts:27]: fix warning

File: src/objects/Coin.ts [hunk 2]
```diff
@@ -21,10 +21,10 @@ const entityConfig: EntityConfigType = {
   collisionCategory: CollisionCategories.coin,
   animations: [
     {
-      animationKey: 'spin',
+      animationKey: 'idle',
```
Comment (norgeous) [inline src/objects/Coin.ts:24]: something is wrong with entity, only `idle` is a valid key atm

File: src/objects/Coin.ts (hunk 1 above)
Comment (ScottyXIII) [reply src/objects/Coin.ts:27]: fucking dam it array starts at 0 lol 😖 Nice catch

File: src/helpers/googleFont.ts [hunk 3]
```diff
@@ -27,6 +27,7 @@ export const webFontLoader = () => {
   document.getElementsByTagName('head')[0].appendChild(script);
 };
 
+// choose new icons here https://fonts.google.com/icons?icon.set=Material+Icons
```
Comment (ScottyXIII) [inline src/helpers/googleFont.ts:30]: Maybe we could this link to the readme as well?

File: src/objects/entities/Entity.ts [hunk 4]
```diff
@@ -133,8 +133,8 @@ class Entity extends Phaser.GameObjects.Container {
     ) as PhaserMatterImage;
     this.scene.add.existing(this);
 
-    if (this.collisionCategory) {
-      this.gameObject.setCollisionCategory(this.collisionCategory);
+    if (collisionCategory) {
```
Comment (ScottyXIII) [inline src/objects/entities/Entity.ts:136]: We could remove the entity class property `collisionCategory ` now then if we're not using it?

Comment (norgeous) [reply src/objects/entities/Entity.ts:136]: yeah but it public, and i wasnt sure if it was used anywhere

i guess i need to check and report back

File: src/helpers/googleFont.ts (hunk 3 above)
Comment (norgeous) [reply src/helpers/googleFont.ts:30]: added

File: src/objects/entities/Entity.ts (hunk 4 above)
Comment (norgeous) [reply src/objects/entities/Entity.ts:136]: yeah it's used here

https://github.com/ScottyXIII/bigheaded/blob/main/src/objects/entities/Ben3.ts#L21

so cannot be removed from being a class property

**Review by ScottyXIII — APPROVED** [review]

**Pull request merged**